<launch>
        <param name="robot_description" command="cat $(find al5d_simulation)/config/model.urdf" />
        <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
        <arg name="rvizconfig" default="$(find al5d_simulation)/config/urdf.rviz" />
        
        <node name="lynx_motion_simulator" pkg="al5d_simulation" type="LynxMotionSimulator" output="screen"  args="0 0 0" /> 
        <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" /> -->
</launch>


